sdcm/utils/health_checker.py

```diff
@@ -268,7 +268,10 @@ def check_schema_agreement_in_gossip_and_peers(node, retries: int = CHECK_NODE_H
 def check_group0_tokenring_consistency(group0_members: list[dict[str, str]],
                                        tokenring_members: list[dict[str, str]],
                                        current_node) -> HealthEventsGenerator:
-    LOGGER.debug("Check all nodes in group0 are voters")
+    if not group0_members:
+        LOGGER.info("Raft feature is disabled")
```

Comment:
Should be debug, we don't need this on output every single health check

Reply:
changed to debug